web/main.ts

```diff
@@ -2324,15 +2358,30 @@ class GitGraphView {
 
 	/* Commit Details View */
 
-	public loadCommitDetails(commitElem: HTMLElement) {
+	public loadCommitDetails(commitElem: HTMLElement, parentIndex: number) {
 		const commit = this.getCommitOfElem(commitElem);
-		if (commit === null) return;
+		if (commit === null) {
+			this.showErrorOnNonLoadedCommit();
+			return;
+		}
 
 		this.closeCommitDetails(false);
 		this.saveExpandedCommitLoading(parseInt(commitElem.dataset.id!), commit.hash, commitElem, null, null);
 		commitElem.classList.add(CLASS_COMMIT_DETAILS_OPEN);
 		this.renderCommitDetailsView(false);
-		this.requestCommitDetails(commit.hash, false);
+		this.requestCommitDetails(commit.hash, false, parentIndex);
+	}
+
+	private showErrorOnNonLoadedCommit() {
+		const actionName = this.moreCommitsAvailable ? 'Load More Commits' : null;
+
+		dialog.showError('The commit could not be found in the loaded commits.<br>' +

```

Comment:
The first argument of `dialog.showError` should be a short message that fits on a single line (e.g. "Unable to find Commit"), and the second argument (currently set to `null` on line 2384) should be the body of the error message. This would make it consistent with all the other error dialogs.